### PR TITLE
Update Developer Guide

### LINKED ISSUE #103: Update DG

### PR DESCRIPTION
**Pull request by Aishwarya-Hariharan-Iyer**
Adds user stories for client commands and for find commands.
Closes #103

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -510,7 +510,132 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-**Use case: UC11 - View list of commands**
+**Use case: UC11 - Add client**
+
+**MSS**
+
+1.  User requests to add a client.
+2.  DevEnable adds the client to the client list.
```
Comment (ConradLew) [inline docs/DeveloperGuide.md:518]: Client list mentioned in this manner seems to suggest internal details of the system - refer to [AB3 developer guide use cases](https://se-education.org/addressbook-level3/DeveloperGuide.html#use-cases) to see how it is circumvented

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -510,7 +510,132 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. The user makes an error in writing the request.
```
Comment (ConradLew) [inline docs/DeveloperGuide.md:524]: I'm not sure if error handling should be considered under extensions, maybe you can clarify in the forum or otherwise

Comment (ConradLew) [reply docs/DeveloperGuide.md:524]: Refer to [here](https://se-education.org/addressbook-level3/DeveloperGuide.html#use-cases) if you would like a better idea of what to do, the use cases I put up in the first iteration may not be up to par

**Review by ConradLew — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Aishwarya-Hariharan-Iyer) [reply docs/DeveloperGuide.md:518]: Understood. Tysm!

**Review by Donovan9617 — APPROVED** [review]

**Pull request merged**